lib/reversetunnel/seek/seek.go

```diff
@@ -369,13 +375,28 @@ func (p *proxyGroup) releaseProxy(t time.Time, principals ...string) (ok bool) {
 
 func (p *proxyGroup) resolveName(principals []string) string {
 	for _, name := range principals {
+		if len(name) >= 36 {
```

Comment:
What's the significance of 36?

Reply:
The `uuid.Parse` function only accepts the hyphenated form of uuid encoding, which is 36 characters.  I'll rework into clearly labeled constant.